test/unit/file.test.js

```diff
@@ -0,0 +1,110 @@
+'use strict';
+
+const assert = require('assert');
+const path = require('path');
+const mockFs = require('mock-fs');
+const sinon = require('sinon');
+
+const { readFile, writeFile, readJson, writeJson } = require('../../lib/file');
+
+const username = 'foo';
+const token = '123456789';
+const fileContent =
+  `{
+    "username": "${username}",
+    "token": "${token}"
+  }`;
+
+const mockDirName = 'files';
+const mockFileName = 'file.json';
+const filePath = path.join(mockDirName, mockFileName);
+
+const newUsername = 'bar';
+const newToken = 'asdfghj';
+
+describe('file', () => {
+  beforeEach(() => {
+    mockFs({
+      [mockDirName]: {
```

Comment:
Right here same comments as above, since the values do not change there is no need to reinit the fs mock so this should only be done once right.